cmd/workerpodautoscaler/run.go

```diff
@@ -63,6 +64,7 @@ func (v *runCmd) new() *cobra.Command {
 		"namespace",
 	}
 
+	flags.Int("scale-down-delay-after-last-scale-activity", 600, "scale down delay after last scale up or down in seconds")
 	flags.Int("resync-period", 20, "sync period for the worker pod autoscaler")
```

Comment:
We should update this description to reflect the current behavior of `resync-period`